
syntax = "proto3";

package google.protobuf.testing;

message DefaultValueTest {
  double double_value = 1;
  repeated double repeated_double = 2;
  float float_value = 3;
  int64 int64_value = 5;
  uint64 uint64_value = 7;
  int32 int32_value = 9;
  uint32 uint32_value = 11;
  bool bool_value = 13;
  string string_value = 15;
  bytes bytes_value = 17 [ctype = CORD];

  enum EnumDefault {
    ENUM_FIRST = 0;
    ENUM_SECOND = 1;
    ENUM_THIRD = 2;
  }
  EnumDefault enum_value = 18;
}
